packages/cli/src/commands/setup/auth/__tests__/__templates__/AppWithCustomRedwoodApolloProvider.template

```diff
@@ -0,0 +1,19 @@
+import { FatalErrorBoundary, RedwoodProvider } from '@redwoodjs/web'
```

Comment:
Instead of putting this file (and the one below) in `__templates__` we typically name that folder `fixtures`. Could you please rename the folder?

Reply:
yeap! is the file extension ok? wanted to avoid `.tsx` in case it tried to compile

Reply:
Actually, could you try `.js`?

Reply:
`AppWithCustomRedwoodApolloProvider.template.js` ?

Reply:
I was thinking just .js, without template